packages/flutter_tools/lib/src/build_system/targets/shader_compiler.dart

```diff
@@ -81,6 +82,23 @@ class ShaderCompiler {
       );
     }
 
+    // Copy the .iplr file to the output path.
+    try {
+      _fs.file('$outputPath.iplr').copySync(outputPath);
+    } on FileSystemException catch (error) {
+      throw ShaderCompilerException._(
+        'Shader compilation of "${input.path}" to "$outputPath" '
+        'failed with a file system error: $error.',
+      );
+    }
+
+    try {
+      _fs.file('$outputPath.iplr').deleteSync();
```

Comment:
I think we have a safe delete helper? Though its not exactly harmless, they end up bundled into the app

Reply:
Replaced with `ErrorHandlingFileSystem.deleteIfExists`, which will `throwToolExit` if the delete fails.